src/util/handlebars/handlebar-helpers.ts

```diff
@@ -25,6 +25,11 @@ export const registerHandlebarsHelpers = () => {
 
 	hbs.registerHelper("isNotConnected", (syncStatus) => syncStatus == null);
 
+	hbs.registerHelper("setSubscriptionUrl", (appId, uuid, installationId) => appId
+		? `/github/subscriptions/${installationId}/ghes?appId=${appId}&uuid=${uuid}`
```

Comment:
Do we need this `ghes`? I don't see any use of this additional parameter.

Reply:
It's in the router `GithubSubscriptionRouter.route("/:installationId/:ghes?")`

Reply:
But its not used inside `GithubSubscriptionGet` nor `GithubSubscriptionDelete`. Won't it still work if we remove this from the route?

Reply:
It's needed to define the query params.

Reply:
Can't we do `/github/subscriptions/${installationId}?appId=${appId}&uuid=${uuid}` ? The query parameters are pulled in the exact same manner. 
Its just that word `ghes` brings up a bit of confusion cause that parameter itself is not used inside any method.